### ISSUE TITLE
Collect information from all sermon series pages

### ISSUE THREAD

**Issue by isaacjwilliams** [wfth/collector#11]

**Comment by isaacjwilliams** [comment]
See 2066af6b98342814e380803a6063cc41d7561c2e

**Comment by aiwilliams** [comment]
In your commit you can do this:
```
Collect from all series pages + handle duplicate series/sermons

Closes #11
```

Then you do not have to comment or click the Close button on the site!

**Comment by isaacjwilliams** [comment]
@aiwilliams I know, I know. I just forget to. :grimacing:
